loggingservice/api/utils/rest_handlers.py

```diff
@@ -29,6 +29,15 @@ def success_response_only_status_code(status_code, msg):
 
     return make_response(resp)
 
+def return_version(version):
+    message = {
```

Comment:
This will need to be changed to match the text/plain response type. Thanks.

Reply:
This is not being used. I removed the method

Reply:
Okay. Thanks.